src/components/navbar/navbar.scss

```diff
@@ -22,13 +27,21 @@
     font-weight: bold;
     color: var(--selected-color, var(--foreground-color));
 
-    // Scegli lo stile delle icone, e dello sfondo delle icone quando vengono selezionate
+    // Scegli lo stile dello sfondo delle icone quando vengono selezionate
```

Comment:
Non descriptive comment. Try to use different words and explain better what that style is for.